app/controllers/aws_texts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class AwsTextsController < ApplicationController
   def index
-    @aws_texts = AwsText.all
+    @aws_texts = AwsText.all.page(params[:page])
```

Comment:
こちらは動画教材ではありませんので修正は不要です。

```suggestion
@aws_texts = AwsText.all
```